.gitignore

```diff
@@ -42,4 +42,15 @@ node_modules/
 build/
 
 # Test
-coverage/
\ No newline at end of file
+coverage/
+
+# lock files
+yarn.lock
+package-lock.json
+pnpm-lock.yaml
```

Comment:
pnpm-lock should not be ignored 🤔

Reply:
I had the same thing in mind, but check this out:

https://github.com/prezly/slate/pull/412#discussion_r1195210573

And another thing is when we're omitting some dependencies the lock file changes which results on different lock files and it's hard to maintain that way.

Reply:
Oh, that's interesting. It would be nice to add [this link](https://github.com/sindresorhus/ama/issues/479#issuecomment-310661514) in a comment here to prevent confusion in the future.